packages/expo-screen-orientation/android/src/main/java/expo/modules/screenorientation/ScreenOrientationModule.java

```diff
@@ -0,0 +1,270 @@
+package expo.modules.screenorientation;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.pm.ActivityInfo;
+import android.util.DisplayMetrics;
+import android.view.Surface;
+import android.view.WindowManager;
+
+import org.unimodules.core.ExportedModule;
+import org.unimodules.core.ModuleRegistry;
+import org.unimodules.core.Promise;
+import org.unimodules.core.errors.InvalidArgumentException;
+import org.unimodules.core.interfaces.ActivityProvider;
+import org.unimodules.core.interfaces.ExpoMethod;
+import org.unimodules.core.interfaces.LifecycleEventListener;
+import org.unimodules.core.interfaces.services.UIManager;
+
+public class ScreenOrientationModule extends ExportedModule implements LifecycleEventListener {
+  private ActivityProvider mActivityProvider;
+  static final String ERR_SCREEN_ORIENTATION = "ERR_SCREEN_ORIENTATION";
+
+  public ScreenOrientationModule(Context context) {
+    super(context);
+  }
+
+  @Override
+  public String getName() {
+    return "ExpoScreenOrientation";
+  }
+
+  @Override
+  public void onCreate(ModuleRegistry moduleRegistry) {
+    mActivityProvider = moduleRegistry.getModule(ActivityProvider.class);
+    moduleRegistry.getModule(UIManager.class).registerLifecycleEventListener(this);
+  }
+
+  @Override
+  public void onHostResume() {
+
+  }
+
+  @Override
+  public void onHostPause() {
+
+  }
+
+  @Override
+  public void onHostDestroy() {
+
+  }
```

Comment:
Shouldn't we do something in these methods? If no, do we need to register lifecycle event listener? 🤔